spec/lib/miq_expression_spec.rb

```diff
@@ -366,6 +397,24 @@
 
     describe "integration" do
       context "date/time support" do
+        it "finds the correct instances for an gt expression with a dynamic integer field" do
+          _vm1 = FactoryGirl.create(:vm_vmware, :memory_reserve => 1, :cpu_reserve => 2)
+          vm2 = FactoryGirl.create(:vm_vmware, :memory_reserve => 2, :cpu_reserve => 1)
+          filter = MiqExpression.new(">" => {"field" => "Vm-memory_reserve", "value" => "Vm-cpu_reserve"})
+          result = Vm.where(filter.to_sql.first)
+          expect(result).to eq([vm2])
+        end
+
+        # it "finds the correct instances for an gt expression with a custom attribute dynamic integer field" do
```

Comment:
@lpichler This test shouldn't pass as is because we cant sql query based on custom_attribute, but how can i test that this condition works?